### ISSUE TITLE
Failing accessibility tests for v0.90

### ISSUE THREAD

**Issue by petdance** [petdance/tidyp#1]
$ cd test $ sh ./testaccess.sh

Testing 1-1-1-1
Testing 1-1-1-2
Testing 1-1-1-3
Testing 1-1-1-4
Testing 1-1-1-10
Testing 1-1-2-1
Testing 1-1-2-2
Testing 1-1-2-3
Testing 1-1-3-1
Testing 1-1-4-1
Testing 1-1-5-1
Testing 1-1-6-1
Testing 1-1-6-2
Testing 1-1-6-3
Testing 1-1-6-4
Testing 1-1-6-5
Testing 1-1-6-6
Testing 1-1-8-1
Testing 1-1-9-1
Testing 1-1-10-1
Testing 1-1-12-1
Testing 1-2-1-1
Testing 1-4-1-1
Testing 1-5-1-1
Testing 2-1-1-1
Testing 2-1-1-2
Testing 2-1-1-3
Testing 2-1-1-4
Testing 2-1-1-5
Testing 2-2-1-1
Testing 2-2-1-2
Testing 2-2-1-3
Testing 2-2-1-4
Testing 3-2-1-1
Testing 3-3-1-1
Testing 3-5-1-1
Testing 3-5-2-1
Testing 3-5-2-2
Testing 3-5-2-3
Testing 3-6-1-1
Testing 3-6-1-2
Testing 3-6-1-4
Testing 4-3-1-1
Testing 4-3-1-2
Testing 5-1-2-1
Testing 5-1-2-2
Testing 5-1-2-3
Testing 5-2-1-1
Testing 5-2-1-2
--- test '5.2.1.2' not detected in file './accessTest/5-2-1-2.html'

Accessibility Checks: Version 0.1

./accessTest/5-2-1-2.html:8:1: Access: [5.1.2.3]: data
missing row/column headers (1 row). Info: Doctype given is "-//W3C//DTD HTML 4.0 Transitional//EN"
Info: Document content looks like HTML 4.01 Strict
No warnings or errors were found.

To learn more about HTML Tidy see http://tidy.sourceforge.net
Please fill bug reports and queries using the "tracker" on the Tidy web site.
Additionally, questions can be sent to html-tidy@w3.org
HTML and CSS specifications are available from http://www.w3.org/
Lobby your company to join W3C, see http://www.w3.org/Consortium
Testing 5-3-1-1
Testing 5-4-1-1
Testing 5-5-1-1
Testing 5-5-1-2
Testing 5-5-1-3
--- test '5.5.1.3' not detected in file './accessTest/5-5-1-3.html'

Accessibility Checks: Version 0.1

./accessTest/5-5-1-3.html:1:1: Access: [3.3.1.1]: use style sheets to control presentation. ./accessTest/5-5-1-3.html:3:1: Access: [13.2.1.1]: Metadata missing. ./accessTest/5-5-1-3.html:7:1: Access: [5.5.1.2]:
summary invalid (null). ./accessTest/5-5-1-3.html:7:1: Access: [5.1.2.1]: data
missing row/column headers (all). Info: Doctype given is "-//W3C//DTD HTML 4.0 Transitional//EN"
Info: Document content looks like HTML 4.01 Strict
No warnings or errors were found.

To learn more about HTML Tidy see http://tidy.sourceforge.net
Please fill bug reports and queries using the "tracker" on the Tidy web site.
Additionally, questions can be sent to html-tidy@w3.org
HTML and CSS specifications are available from http://www.w3.org/
Lobby your company to join W3C, see http://www.w3.org/Consortium
Testing 5-5-1-6
Testing 5-5-2-1
Testing 5-6-1-1
Testing 5-6-1-2
Testing 5-6-1-3
--- test '5.6.1.3' not detected in file './accessTest/5-6-1-3.html'

Accessibility Checks: Version 0.1

./accessTest/5-6-1-3.html:1:1: Access: [3.3.1.1]: use style sheets to control presentation. ./accessTest/5-6-1-3.html:3:1: Access: [13.2.1.1]: Metadata missing. ./accessTest/5-6-1-3.html:8:1: Access: [5.4.1.1]: invalid markup used in layout
. ./accessTest/5-6-1-3.html:11:13: Access: [5.6.1.2]:
header abbreviations invalid (null). ./accessTest/5-6-1-3.html:11:72: Access: [5.6.1.2]:
header abbreviations invalid (null). Info: Doctype given is "-//W3C//DTD HTML 4.0 Transitional//EN"
Info: Document content looks like HTML 4.01 Strict
No warnings or errors were found.

To learn more about HTML Tidy see http://tidy.sourceforge.net
Please fill bug reports and queries using the "tracker" on the Tidy web site.
Additionally, questions can be sent to html-tidy@w3.org
HTML and CSS specifications are available from http://www.w3.org/
Lobby your company to join W3C, see http://www.w3.org/Consortium
Testing 6-1-1-1
Testing 6-1-1-2
Testing 6-1-1-3
Testing 6-2-1-1
Testing 6-2-2-1
Testing 6-2-2-2
Testing 6-2-2-3
Testing 6-3-1-1
Testing 6-3-1-2
Testing 6-3-1-3
Testing 6-3-1-4
Testing 6-5-1-1
Testing 6-5-1-2
Testing 6-5-1-3
Testing 6-5-1-4
Testing 7-1-1-1
Testing 7-1-1-2
Testing 7-1-1-3
Testing 7-1-1-4
Testing 7-1-1-5
Testing 7-2-1-1
Testing 7-4-1-1
Testing 7-5-1-1
Testing 8-1-1-1
Testing 8-1-1-2
Testing 8-1-1-3
Testing 8-1-1-4
Testing 9-1-1-1
Testing 9-3-1-1
Testing 9-3-1-2
Testing 9-3-1-3
Testing 9-3-1-4
Testing 9-3-1-5
Testing 9-3-1-6
Testing 10-1-1-1
Testing 10-1-1-2
Testing 11-2-1-1
Testing 11-2-1-2
Testing 11-2-1-3
Testing 11-2-1-4
Testing 11-2-1-5
Testing 11-2-1-6
Testing 11-2-1-7
Testing 11-2-1-8
Testing 11-2-1-9
Testing 11-2-1-10
Testing 12-1-1-1
Testing 12-1-1-2
Testing 12-1-1-3
Testing 12-4-1-1
Testing 12-4-1-2
Testing 12-4-1-3
Testing 13-1-1-1
Testing 13-1-1-2
Testing 13-1-1-3
Testing 13-1-1-4
Testing 13-2-1-1
Testing 13-2-1-3
Testing 13-10-1-1

===================8<------------------------------------

$ uname -a Linux gnurf 2.6.31-17-generic #54-Ubuntu SMP Thu Dec 10 16:20:31 UTC 2009 i686 GNU/Linux

libtidy compiled with no visible warnings or errors.
